config-connector/solutions/apps/helm/wordpress/README.md

```diff
@@ -0,0 +1,55 @@
+# Wordpress
+
+==================================================
+
+## NAME
+
+  wordpress
+
+## SYNOPSIS
+
+The WordPress application demonstrates how you can configure a WordPress site powered by GCP MySQL database and using Workload Identity for authentication.
+
+## CONSUMPTION
+
+  ```bash
+   helm install ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID]
```

Comment:
Shouldn't there be a [NAME] parameter here?

Reply:
Also, why is the install command both here and in the USAGE section?

Reply:
Agreed with both. I realized consumption section is about preparing what's needed for usage. I changed this section to instead clone and go to the right folder.

Reply:
Agreed with both. I realized, consumption section is about preparing for the steps in Usage section. Added different steps.